module/applications/item/summoning-config.mjs

```diff
@@ -0,0 +1,135 @@
+/**
+ * Application for configuring summoning information for an item.
+ */
+export default class SummoningConfig extends DocumentSheet {
+
+  /** @inheritDoc */
+  static get defaultOptions() {
+    return foundry.utils.mergeObject(super.defaultOptions, {
+      classes: ["dnd5e", "summoning-config"],
+      dragDrop: [{ dragSelector: ".drag-bar", dropSelector: "form" }],
```

Comment:
Not sure if this `dragSelector` is needed anymore. Fine to leave it if `dragDrop` requires it.